dzVents update to 3.0.8

rwaaren opened the pull request:
- Allow IPv6 ::1 as localhost in domoticz settings
- Fixed bug that occurred when using a decimal number in afterSec (openURL and emitEvent)
- Implement optional use of parsetrigger parm in setValues to trigger any subsequent eventscripts
- Updated round.utils to correctly handle negative numbers and round to zero decimals


```
========= domoticz V2020.2 (12100), dzVents V3.0.8 +========================== Tests ==============================+
  time  | test-script                              | expected | tests | result  | successful |  failed  | seconds  |
===================================================+===============================================================+
00:00:02  Device                                        1        115      OK            115         0      0.2779
00:00:03  Domoticz                                      1         74      OK             74         0      0.2796
00:00:03  EventHelpers                                  1         32      OK             32         0      0.2532
00:00:03  EventHelpersStorage                           1         50      OK             50         0      0.2378
00:00:04  HTTPResponse                                  1          6      OK              6         0      0.0147
00:00:04  Lodash                                        1        100      OK            100         0      0.1514
00:00:04  ScriptdzVentsDispatching                      1          2      OK              2         0      0.0652
00:00:04  TimedCommand                                  1         46      OK             46         0      0.1078
00:00:04  Time                                          3        340      OK            340         0      0.7092
00:00:05  Utils                                         1         27      OK             27         0      0.0490
00:00:05  Variable                                      1         15      OK             15         0      0.0326
00:00:05  ContactDoorLockInvertedSwitch                 3          2      OK              2         0      3.0669
00:00:08  DelayedVariableScene                          8          2      OK              2         0      7.0691
00:00:15  EventState                                   19          2      OK              2         0     18.0789
00:00:34  Integration                                  34        222      OK            222         0     28.6301
00:01:02  SelectorSwitch                               10          2      OK              2         0      9.2067
00:01:18  SystemAndCustomEvents                         1          7      OK              7         0      0.0363


Total tests: 1044
Wed 27 May 2020 04:02:54 PM CEST; dzVents version 3.0.8 tested without erors after 00:01:18

```

rwaaren committed:
dzVents update to 3.0.8

- Allow IPv6 ::1 as localhost in domoticz settings
- Fixed bug that occurred when using a decimal number in afterSec (openURL and emitEvent)
- Implement optional use of parsetrigger parm in setValues to trigger any subsequent eventscripts
- Updated round.utils to correctly handle negative numbers and round to zero decimals

rwaaren commented:
Fix for #4182 and forum posts [roborock dzvents problem](https://www.domoticz.com/forum/viewtopic.php?f=59&t=32849&p=248882)
/ [sudden error](https://www.domoticz.com/forum/viewtopic.php?f=59&t=32824)

gizmocuz commented:
@rwaaren , quick question, why are you putting the output of lua_toboolean into a float ? This function returns 0 or 1, a integer would maybe make more sense ?

rwaaren committed:
Corrected wrong mapping of lua_toboolean value to float; map to int now

Retested successfully after correction

Total tests: 1044
Thu 28 May 2020 12:58:11 PM CEST; dzVents version 3.0.8 tested without erors after 00:01:17

rwaaren commented:
> why are you putting the output of lua_toboolean into a float ? This function returns 0 or 1

You are right, was a leftover of a global copy /paste action. 
Corrected with a follow up commit.

@gizmocuz Thanks for spotting this !
